playbooks/templates/standalone_parameters.yaml.j2

```diff
@@ -1,5 +1,9 @@
 #vi:syntax=yaml
 
+resource_registry:
+  # i need to figure out how to re-enable it without binding error during the deployment...
+  OS::TripleO::Services::Horizon: OS::Heat::None
+
```

Comment:
Presumably this is unrelated?

Incidentally, as long as it doesn't add too much complexity I'd be ok with a flag to disable Horizon. I don't know how much it costs, but we're presumably never going to use it in CI and rarely in dev. I'd stick with keeping it enabled by default, though.